test/functional/outputs/syslog/rfc3164_test.go

```diff
@@ -55,37 +55,9 @@ var _ = Describe("[Functional][Outputs][Syslog] RFC3164 tests", func() {
 		Expect(outputlogs[0]).To(MatchRegexp(expMatch), "Exp to find tag in received message")
 		Expect(outputlogs[0]).To(MatchRegexp(`{"index":.*1,.*"timestamp":.*1,.*"tag_key":.*"rec_tag"}`), "Exp to find the original message in received message")
 	},
-
-		Entry("should use the value from the record and include the message", "$.message.tag_key", "rec_tag", false),
```

Comment:
Same comment here.  Also, let's add a test which demonstrates the bug no longer exists and proves we fixed it